File: models/tests/twobody/make_init.py
import lilguys
import numpy as np

def main():
    p0 = 1
    v0 = 0.3
    x0 = np.array([[p0,0,0], 
        [-p0,0,0]])
    v0 = np.array([[0,v0,0], 
        [0,-v0,0]])

    m = 1

    s = lilguys.Snapshot(x0, v0, m=m)
    s.save("twobody.hdf5")


if __name__ == "__main__":
    main()

File: simulations/tests/twobody/make_init.jl
import LilGuys as lguys

function main()
    p0 = 1
    v0 = 0.3
    x0 = [p0 -p0
          0   0 
          0   0] 

    v0 = [0 0
          v0 -v0
          0 0]

    m = [1, 1]

    s = lguys.Snapshot(x0, v0, m)
    lguys.save("twobody.hdf5", s)
end


main()
